### PR TITLE
doc: add minutes for meeting 23 April 2020

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <michael_dawson@ca.ibm.com>

### CODE REVIEW

File: meetings/2020-04-23.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  * Ruben will write blog post, idea would be to have survey link at the end. 
+
+* http: align with stream.Writable [31818](https://github.com/nodejs/node/pull/31818)
+  * Rich has asked for TSC concensus or not on overriding the objection. In this comment 
+    https://github.com/nodejs/node/pull/31818#issuecomment-618180372
+
+### nodejs/docker-node
+
+* Suggestion for dropping Yarn from Node 14 release [#1238](https://github.com/nodejs/docker-node/issues/1238)
+  * Missed window, already released, does not need discussion.
```
Comment (ljharb) [inline meetings/2020-04-23.md:70]: doesn’t this need discussion for v15, at least?

File: meetings/2020-04-23.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+* V8 Currency
+  * No updates from me either. Will probably start working again on the V8 8.3 update PR next week.
+
+* Core Promise APIs
+  * No update.
+
+* Build Resources: 
+  * nothing new to report this week.
+
+* joyeecheung
```
Comment (mmarchini) [inline meetings/2020-04-23.md:100]:
```suggestion
* Startup Performance
```

File: meetings/2020-04-23.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  * There are two prototypes for including more of the bootstrap process into the startup snapshot, with different designs: nodejs/node#32761 and nodejs/node#32984 . Both are blocked by a V8 issue and there is a WIP in the upstream to fix it .
+
```
Comment (mmarchini) [inline meetings/2020-04-23.md:102]:
```suggestion
```

File: meetings/2020-04-23.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Ruben Bridgewater @BridgeAR (TSC)
+* Sam Roberts @sam-github (TSC)
+
+## Agenda
+
+### Announcements
+
+* Myles, Node.js 14 went out this week !   Will be our next LTS release.
```
Comment (cjihrig) [inline meetings/2020-04-23.md:22]:
```suggestion
* Myles, Node.js 14 went out this week! Will be our next LTS release.

```

File: meetings/2020-04-23.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+* Board meeting this, week.
+  * James asked about DCO side of things.
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* openssl security release 1.1.1g - vulnerability HIGH [#32846](https://github.com/nodejs/node/issues/32846)
+  * Turns out it does not affect Node.js so no need to discussion.
```
Comment (cjihrig) [inline meetings/2020-04-23.md:34]:
```suggestion
  * Turns out it does not affect Node.js so no need for discussion.

```

File: meetings/2020-04-23.md (hunk 1 above)
Comment (Trott) [reply meetings/2020-04-23.md:70]: > doesn’t this need discussion for v15, at least?

/ping @nschonni What do you think? Too early to discuss that?

File: meetings/2020-04-23.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* James Snell @jasnell (TSC)
+* Matheus Marchini @mmarchini (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Ruben Bridgewater @BridgeAR (TSC)
+* Sam Roberts @sam-github (TSC)
+
```
Comment (Trott) [inline meetings/2020-04-23.md:17]:
```suggestion
* Rich Trott @Trott (TSC)

```

File: meetings/2020-04-23.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+  * Original plan was to do survey
+  * Matheus: Only people who are affected by the deprecation warning will be broken. With 
+    proposed change handler would be enough so no command line flag required. One thing
+    is that any module can add handler which would cause the override.
+  * Need to integrate discuss in the issue.
+  * Ruben, prefer that we still have survey. 
+  * Ruben will write blog post, idea would be to have survey link at the end. 
+
+* http: align with stream.Writable [31818](https://github.com/nodejs/node/pull/31818)
+  * Rich has asked for TSC concensus or not on overriding the objection. In this comment
```
Comment (Trott) [inline meetings/2020-04-23.md:67]:
```suggestion
  * Rich has asked for TSC consensus or not on overriding the objection. In this comment 
```

File: meetings/2020-04-23.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+  * Original plan was to do survey
+  * Matheus: Only people who are affected by the deprecation warning will be broken. With 
+    proposed change handler would be enough so no command line flag required. One thing
+    is that any module can add handler which would cause the override.
+  * Need to integrate discuss in the issue.
+  * Ruben, prefer that we still have survey. 
+  * Ruben will write blog post, idea would be to have survey link at the end. 
+
+* http: align with stream.Writable [31818](https://github.com/nodejs/node/pull/31818)
+  * Rich has asked for TSC consensus or not on overriding the objection. In this comment
```
Comment (Trott) [inline meetings/2020-04-23.md:67]:
```suggestion
  * Rich has asked for TSC consensus or not on overriding the objection. See 
```

File: meetings/2020-04-23.md (hunk 1 above)
Comment (nschonni) [reply meetings/2020-04-23.md:73]: Probably doesn't hurt. I just mentioned over on the other issue, that there are long standing discussions on other threads about it. If the TSC isn't going to weight it, I don't really want to take another beating from the Yarn folks right now.

PS: There is some WG vs Colloborator divide on the docker-node project that I don't exactly understand, and the push to remove it was more from the Collaborators than anyone on the WG

Comment (ljharb) [reply meetings/2020-04-23.md:73]: imo the decision to include it initially should have had much broader consensus, beyond the WG, perhaps even the TSC.

**Pull request merged**